### PR TITLE
Add OCPU.pmStaff

### PR DESCRIPTION
**Pull request by zellman01**

### CODE REVIEW

**Comment by zellman01** [comment]
For the record, the reason this was "put up" earlier than #31 is because my computer's time was off

File: chat-plugins/ocpuusers.js [hunk 1]
```diff
@@ -228,6 +228,16 @@ OCPU.messageSeniorStaff = function (message, pmName, from) {
 		}
 	});
 };
+
+OCPU.pmStaff = function (message, pmName, from) {
+	pmName = (pmName ? pmName : '~Staff PM [DO NOT REPLY]');
+	from = (from ? ' (PM from ' + from + ')' : "");
+	Users.users.forEach(curUser => {
+		if (curUser === '%' || curUser === '@' || curUser === '&' || curUser === '~') {
```
Comment (DeathlyPlays) [inline chat-plugins/ocpuusers.js]: Use ``if (user.isStaff)`` imo

Comment (zellman01) [reply chat-plugins/ocpuusers.js]: On Line 236?

Comment (DeathlyPlays) [reply chat-plugins/ocpuusers.js]: Yea

**Pull request merged**